src/Infra/HTTP/AuthHttpService.ts

```diff
@@ -0,0 +1,23 @@
+import { KeyParams } from '@standardnotes/auth'
+import { inject, injectable } from 'inversify'
+import { SuperAgentStatic } from 'superagent'
+import TYPES from '../../Bootstrap/Types'
+import { AuthHttpServiceInterface } from '../../Domain/Auth/AuthHttpServiceInterface'
+
+@injectable()
+export class AuthHttpService implements AuthHttpServiceInterface {
+  constructor (
+    @inject(TYPES.HTTPClient) private httpClient: SuperAgentStatic,
+    @inject(TYPES.AUTH_SERVER_URL) private authServerUrl: string,
+  ) {
+  }
+
+  async getUserKeyParams(email: string, authenticated: boolean): Promise<KeyParams> {
+    const keyParamsResponse = await this.httpClient
+      .get(`${this.authServerUrl}/users/params`)
```

Comment:
So I'm not sure how this works with the new server, but with the Ruby server, getting auth params for a user requires their 2FA token. Is this the same here? You might be able to make an exception for internal requests.

Reply:
This service basically calls Auth Server to execute the code which was initially in here: https://github.com/standardnotes/auth/blob/develop/src/Controller/UsersController.ts#L91-L108

Reply:
Hmm but in that code snippet I'm not seeing 2FA mention. We have this in SS Ruby: https://github.com/standardnotes/syncing-server/blob/develop/app/controllers/api/auth_controller.rb#L238